index.html

```diff
@@ -10918,22 +11181,37 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
       </tbody>
       <tbody>
         <tr>
-          <td><dfn><code>icecandidateerror</code></dfn></td>
-          <td><code><a>RTCIceGathererIceErrorEvent</a></code></td>
-          <td>The <code><a>RTCIceGatherer</a></code> object has experienced an ICE
-          gathering failure (such as an authentication failure with TURN
-          credentials).</td>
+          <td><code>error</code></td>
+          <td><code><a>ErrorEvent</a></code></td>
+          <td>The <code><a>RTCQuicTransport</a></code> object has encountered an error.</td>
         </tr>
         <tr>
           <td><code>statechange</code></td>
           <td><code><a>Event</a></code></td>
-          <td>The <code><a>RTCIceGathererState</a></code> changed.</td>
+          <td>The <code><a>RTCQuicTransportState</a></code> changed.</td>
         </tr>
         <tr>
-          <td><code>icecandidate</code></td>
-          <td><code><a>RTCIceGatherer</a></code></td>
-          <td>A new <code><a>RTCIceGatherCandidate</a></code> is made available to the
-          script.</td>
+          <td><dfn><code>quicstream</code></dfn></td>
+          <td><code><a>RTCQuicStreamEvent</a></code></td>
+          <td>A new <code><a>RTCQuicStream</a></code> is dispatched to the script in
+          response to the remote peer creating a QUIC stream.</td>
+        </tr>
+      </tbody>
+    </table>
+    <p>The following events fire on <code><a>RTCQuicStream</a></code> objects:</p>
+    <table style="border-width:0; width:60%" border="1">
+      <tbody>
+        <tr>
+          <th>Event name</th>
+          <th>Interface</th>
+          <th>Fired when...</th>
```

Comment:
I assume this is TBD?

Reply:
The only event for RTCQuicStream is the statechange event.

Reply:
Oh, it's a table :).  Nevermind.